handbook/about_dbt_labs.md

```diff
@@ -1,17 +1,28 @@
 # The dbt Labs Employee Handbook
 
+| Maintained by |
+|---|
+| Meg Colón |
+| Leigh Staub |
+
 ## Welcome to dbt Labs
-Hello and Welcome to dbt Labs. We’re thrilled to have you as a member of the team!
+Hello and welcome to dbt Labs. We’re thrilled to have you as a member of the team!
 
-This handbook is intended to answer many of the questions you might have after joining, and should serve as a valuable resource throughout your employment at dbt Labs. Read on to learn about important company policies and procedures, as well as the commitments of this company to our employees.
+This handbook is intended to answer your questions as a new employee and should serve as a valuable resource throughout your employment at dbt Labs. Read on to learn about important company policies and procedures, as well as the commitments of this company to our employees.
 
 ## Overview
-#### [Mission](mission.md) and [Values](values.md)
-#### [Our Distributed Work Experience](#Our-Distributed-Work-Experience)
-#### [Working and Growing Here](/handbook/working_and_growing_here.md)
-#### [Compensation](/handbook/compensation.md)
-#### [Benefits and Perks](/handbook/benefits.md)
-#### [Training and Policies]
+[Mission](/mission.md) and [Values](/values.md)
+
+[Our Distributed Work Experience](#our-distributed-work-experience)
+
+[Working and Growing Here](/handbook/working_and_growing_here.md)
+
+[Compensation](/handbook/compensation.md)
+
+[Benefits and Perks](/handbook/benefits.md)
+
+[Policies and Training](/handbook/policies_and_training.md)
+
 
 
 ## [Our Distributed Work Experience](https://www.notion.so/dbtlabs/How-we-think-about-the-Distributed-Experience-5edaf90eafad420b9a05ffc03a3cc5c7?pvs=4)
```

Comment:
Should this link to the Notion page if everything is below?

Reply:
You can ignore the links to Notion pages on the headers. I added them everywhere for now to easily point back to where it is in Notion for this editing process. It will also help when we need to delete content from Notion to know what needs deleted and what needs redirected to other content.

We will remove these Notion links once the handbook is in a more final state and we clean up Notion.